client/src/modules/employees/registry/search.modal.js

```diff
@@ -33,6 +33,7 @@ function EmployeeRegistryModalController(ModalInstance, Functions, Grades, bhCon
 
   // assign default filters
   if (filters.limit) {
+    vm.searchQueries.limit = filters.limit;
```

Comment:
Once you change this code to use `vm.defaultQueries.limit`, you can remove this line.